libs/bank-vault/config.jsonnet

```diff
@@ -0,0 +1,16 @@
+// libs/<name>/config.jsonnet
+local config = import 'jsonnet/config.jsonnet';
+
+config.new(
+  name='bank-vault-operator',
+  specs=[
+    {
+      output: 'v0.4.1',
```

Comment:
i don't remember ... why is tihs 0.4.1 if the version of bank vaults is 1.15.3 ?